Response/RoutesResponse.php

```diff
@@ -56,6 +56,8 @@ public function getRoutes()
                 'defaults'     => $defaults,
                 'requirements' => $route->getRequirements(),
                 'hosttokens'   => method_exists($compiledRoute, 'getHostTokens') ? $compiledRoute->getHostTokens() : array(),
+                'methods'      => $route->getMethods(),
```

Comment:
> it isn't, but as this could be defined in the requirements, similar to the schemes, before Symfony 3, I migrated that as well so someone that uses it could make it work.

Ok, good point :+1:

> I could remove this or add it to the prototype in the JS file as well.

Imo you should add it to the JS prototype :)